config/resources/concept-domain.json

```diff
@@ -88,6 +88,20 @@
       },
       "features": ["include-uri"],
       "new-resource-base": "http://themis.vlaanderen.be/id/concept-schema/"
+    },
+    "document-types": {
+      "name": "document-type",
+      "class": "ext:DocumentType",
+      "super": ["concept"],
+      "relationships": {
+        "document-containers": {
+          "predicate": "dct:type",
+          "target": "document-container",
+          "inverse": true,
+          "cardinality": "many"
+        }
+      },
+      "new-resource-base": "http://themis.vlaanderen.be/id/concept/"
```

Comment:
Why not make this `http://themis.vlaanderen.be/id/concept/document-type`?

Reply:
Did this because Niels and Aad said the resource base could be reused, and out of fear that the inheritance of the generic Concept would not work properly for the existing links if I modified this. 
Plus there are still DocumentTypes that have old URIs. We'll look at this when we fix those